core/storage/compacter/lookback.hpp

```diff
@@ -19,8 +19,9 @@ namespace fc::storage::compacter {
   using vm::actor::code::Code;
 
   constexpr bool anyOf(
```

Comment:
What about?

`template <typename TCode, typename... Args>`
`bool anyOf(TCode code, Args... args) {`
`    return (... || (code == args));`
`}`

Reply:
We need exact argument type and count here

Reply:
It is suitable for your case and you don't need to change it anymore further when new actors are released

Reply:
Not suitable